sound/soc/sof/sof-client.c

```diff
@@ -289,7 +289,7 @@ struct sof_ipc4_fw_module *sof_client_ipc4_find_module_by_uuid(struct sof_client
 {
 	struct snd_sof_dev *sdev = cdev->sdev;
 
-#if IS_ENABLED(CONFIG_SND_SOC_SOF_INTEL_IPC4)
+#ifdef CONFIG_SND_SOC_SOF_INTEL_IPC4
```

Comment:
Move this when `sof_client_ipc4_find_module_by_uuid()` was introduced?

Reply:
use if(IS_ENABLED), it preserves indentation and is nicer to read.

Reply:
> Move this when `sof_client_ipc4_find_module_by_uuid()` was introduced?

Yep, will do, and I'll put the whole function behind ifdefs as you suggested. No sense to define a function if its not supported.